lib/elixir/lib/inspect/algebra.ex

```diff
@@ -88,8 +93,33 @@ defmodule Inspect.Opts do
           pretty: boolean,
           safe: boolean,
           syntax_colors: [{color_key, IO.ANSI.ansidata()}],
-          inspect_fun: (any, t -> Inspect.Algebra.t())
+          inspect_fun: (any, t -> Inspect.Algebra.t()),
+          custom_options: [{atom, any()}]
```

Comment:
This is exactly the definition of `t:keyword/0` :)

```suggestion
          custom_options: keyword
```